implementation/src/test/java/io/smallrye/metrics/exporters/JsonExporterTest.java

```diff
@@ -0,0 +1,40 @@
+package io.smallrye.metrics.exporters;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotNull;
+
+import java.util.Arrays;
+import java.util.List;
+
+import org.eclipse.microprofile.metrics.Meter;
+import org.eclipse.microprofile.metrics.MetricRegistry;
+import org.junit.Test;
+
+import io.smallrye.metrics.MetricRegistries;
+import io.smallrye.metrics.app.MeterImpl;
+
+public class JsonExporterTest {
+
+    @Test
+    public void exportOneMetric_metered() {
+
+        // maybe this should be System.getProperty("line.separator") ?
```

Comment:
I don't think so, if the exporter was using that, then the client would get different data depending on the platform where the server is running, which is not a good thing. Using `\n` in the exporter as well as tests is probably fine.